src/homepage/_posts/2017-10-02-BlogPostInstallingJames.markdown

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title:  "Blog post: Easy and secure James installation"
+date:   2017-10-03 16:13:22 +0200
+categories: james update
+---
+
+In a recent [blog post], Thibaut explain us how to easily set up a James sever on a personal domain.
+
+He gives hints about DNS configuration, SSL configuration, James configuration, and SPF configuration. All of
+it with working examples!
```

Comment:
I am not really good in english but are you sure "All of it with" is correct, for me it sound weird I would have written "All of this"

Reply:
_All those features explained with_

Reply:
I would say "TLS" instead of "SSL"